guides/source/active_storage_overview.md

```diff
@@ -500,7 +500,7 @@ message.video.open do |file|
 end
 ```
 
-It's important to know that the file are not yet available in the `after_create` callback but in the `after_create_commit` only.
+NOTE: It's important to know that the file is not yet available in the `after_create` callback but in the `after_create_commit` only.
```

Comment:
Can you drop the NOTE, does't need highlighting here, since its a simple running sentence.